packages/slack-app/src/routes/learning-collector/messages.ts

```diff
@@ -244,26 +254,36 @@ export async function learnerSummary(learnings: Learning[], members: Member[]) {
 }
 
 async function createUserSection(member: Member, learnings: Learning[]) {
-  let reactionsCount = 0;
-  for (const learning of learnings) {
-    const reactions = await getReactionsFromMessage(learning.shareMessage);
-    reactionsCount =
-      reactionsCount +
-      reactions.reduce((sum, reaction) => sum + reaction.count, 0);
-  }
+  const accumulatedReactions = await getReactionsCount(learnings);
+  const reactionsText = createReactionsText(accumulatedReactions);
   const userLearnings = learnings
     .map(createLearningNumberedListEntry)
     .join("\n\n");
-  const reactionEmoji = getVoteReactions().reactions[0];
   return {
     type: "section",
-    text: markdown(
-      `*<@${member.id}>*\n\n${userLearnings}\n\n:${reactionEmoji}: ${reactionsCount}`
-    ),
+    text: markdown(`*<@${member.id}>*\n\n${userLearnings}\n\n${reactionsText}`),
     accessory: profilePhoto(member),
   };
 }
 
+async function getReactionsCount(learnings: Learning[]) {
+  let accumulatedReactions: Reaction[] = [];
+  for (const { shareMessage } of learnings) {
+    if (!shareMessage.channel) continue;
+    const reactions = await getReactionsFromMessage(shareMessage);
+    accumulatedReactions = reactions.reduce((accumulated, reaction) => {
+      const index = accumulated.findIndex((r) => r.name === reaction.name);
+      if (index >= 0) {
+        accumulated[index].count = +reaction.count;
+      } else {
+        accumulated.push(reaction);
```

Comment:
Mutating accumulated variables inside a reduce statement is a bit off code-smell.